entities/updateData.go

```diff
@@ -0,0 +1 @@
+package entities
```

Comment:
empty file?

Reply:
Plan to add some extra code formatting using struct for all the error messages